KBoard/KBoard/View/WordCustomCell.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  WordCustomCell.swift
+//  KBoard
+//
+//  Created by ParkJunHyuk on 2022/07/24.
+//
+
+import UIKit
+
+class WordCustomCell : UITableViewCell, UITableViewDelegate{
+    
+    static let tableCellId = "TableCellId"
+    
+    lazy var HangleName : UILabel = {
+        let label = UILabel()
+        label.textColor = .black
+        label.font = UIFont.boldSystemFont(ofSize: 25) // 기본 size 가 16
+        return label
+    }()
+    
+    lazy var EnglishName : UILabel = {
+        let label = UILabel()
+        label.textColor = .systemGray2
+        return label
+    }()
+    
+    lazy var favoriteButton : UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(systemName: "star.fill"), for: .normal)
+//        button.tintColor = .secondaryLabel
+        return button
+    }()
+    
+    lazy var RectangleView : UIView = {
+        let view = UIView()
+        view.frame = CGRect(x: 0, y: 0, width: 200, height: 130)
+        view.backgroundColor = UIColor.white
+        
+        view.layer.cornerRadius = 10
+        view.layer.shadowOffset = CGSize(width: 0, height: 0)
+        view.layer.shadowColor = UIColor.gray.cgColor
+        view.layer.shadowOpacity = 0.3
+        view.layer.shadowRadius = 5
+        return view
+    }()
+
+    // 스토리보드로 셀을 작성하게 될 경우 초기화를 해주기 때문에 안해줘도 되지만 코드로 작성하게될 경우 작성해주어야 한다
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        render()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder: ) has not been implemneted")
+    }
+    
+    override func layoutSubviews() {
+         super.layoutSubviews()
+        
+         contentView.frame = contentView.frame.inset(by: UIEdgeInsets(top: 0, left: 16, bottom: 160, right: 16))
+        
+     }
+    
+    func render() {
+        
+        self.addSubview(RectangleView)
+        
+        RectangleView.anchor(top: self.safeAreaLayoutGuide.topAnchor, left: self.safeAreaLayoutGuide.leftAnchor, bottom: self.safeAreaLayoutGuide.bottomAnchor, right: self.safeAreaLayoutGuide.rightAnchor, paddingTop: 10, paddingLeft: 10, paddingBottom: 10, paddingRight: 10)
+
+        
+        self.addSubview(HangleName)
+        
+        HangleName.anchor(top: RectangleView.topAnchor, left: RectangleView.leftAnchor, paddingTop: 20, paddingLeft: 20)
```

Comment:
```suggestion
        self.addSubview(RectangleView)
        RectangleView.anchor(top: self.safeAreaLayoutGuide.topAnchor, left: self.safeAreaLayoutGuide.leftAnchor, bottom: self.safeAreaLayoutGuide.bottomAnchor, right: self.safeAreaLayoutGuide.rightAnchor, paddingTop: 10, paddingLeft: 10, paddingBottom: 10, paddingRight: 10)

        self.addSubview(HangleName)
        HangleName.anchor(top: RectangleView.topAnchor, left: RectangleView.leftAnchor, paddingTop: 20, paddingLeft: 20)
```

확실한 역할 나누어 가독성을 높히는데 공백은 굉장한 도움을 줍니다. 
다만 위의 역할들은 공백을 조금 줄여 묶어서 봐도 괜찮을꺼라 생각하는데 어떤가요??🤔
render 함수 내에 전반적으로 공백이 조금 보이는 것 같아요..!

Reply:
줄바꿈을 1줄로만 변경하였고
`addSubView` 구문과 `anchor` 구문 사에이 있는 줄바꿈을 없에서 보기 더욱 편하게 바꾸었습니다.
반영 완료!